README.md

```diff
@@ -1,2 +1,16 @@
 # hw01
 Repository for homework 01
+
+# Introduction
+
+Hello. My name is Chandler James. I'm a second year PhD student in the political science department. Before graduate school, I worked as a staffer for a US congressman. My research interests include:
+
+* American politics
+* institutions 
+* public opinion
+
+
+You can follow me on *twitter* at this link -> http://twitter.com/chnjms
```

Comment:
This is great! If you're interested in taking linking to the next level, I got the comment in my work to wrap the text in `[]` and `()` to directly link the URL to a text. No necessary, but fun fact. 

For example:

`You can follow me on [*Twitter*](http://twitter.com/chnjms)`

You can follow me on [*Twitter*](http://twitter.com/chnjms)